WordPress/src/main/res/values/strings.xml

```diff
@@ -47,6 +47,9 @@
     <string name="edit_photo_screen_title">Edit Photo</string>
     <string name="notif_detail_screen_title">Notification detail %s</string>
 
+    <!-- default strings -->
+    <string name="wordpress_as_site_title_when_no_title">WordPress</string>
```

Comment:
The resource name feels a bit too specific even though we're actually referring to the WordPress name here, which is kinda generic. What do you think about naming it just `wordpress`?

Also, let's mark it as un-translatable while at it, wdyt?

Reply:
Renamed it and set `translatable="false"`